src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,84 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000, true),
+    THIRD(5, 1_500_000, false),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
+    MISS(0, 0);
+
+    private int countOfMatch;
+    private boolean checkBonus;
+    private int winningMoney;
+
+    private Rank(int countOfMatch, int winningMoney) {
```

Comment:
```suggestion
   Rank(int countOfMatch, int winningMoney) {
```
enum의 생성자에 접근제어자는 불필요합니다. 😀

Reply:
네~!! 삭제 하겠습니다 😃